src/aap_eda/utils/__init__.py

```diff
@@ -0,0 +1,17 @@
+#  Copyright 2024 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+def str_to_bool(value: str) -> bool:
+    return value.lower() in ("yes", "true", "1")
```

Comment:
Is it consistent with other ansible code to allow anything but these three values to be `false`?

Reply:
It is consistent with what we are currently doing here. There is not a specific requirement about it.